aspnetcore/release-notes/aspnetcore-2.2.md

```diff
@@ -96,3 +96,13 @@ For more information, see [the pull request that made this improvement](https://
 ## Additional information
 
 For the complete list of changes, see the [ASP.NET Core 2.2 Release Notes](https://github.com/aspnet/Home/releases/tag/2.2.0).
+
+## Web app templates updated to Bootstrap 4
```

Comment:
There's already a templates section a couple of sections earlier, all we need to do is add the image to that section.